pkg/status/combinedstatus-resolver.go

```diff
@@ -102,6 +103,10 @@ type CombinedStatusResolver interface {
 	//
 	// The returned pointers are expected to be read-only.
 	ResolutionExists(name string) (string, util.ObjectIdentifier, bool)
+
+	// MissingStatusCollectors returns the names of StatusCollector object(s)
+	// that are required by a binding but do not exist.
+	MissingStatusCollectors(bindingName string) []string
```

Comment:
Since the caller is relying on it, I would document in the comment here that the returned list is sorted.

And again, here is a return of a mutable data structure. The comment should document that the caller may mutate the slice and the callee does not cause anything to mutate the slice after return.